doc/manual.rst

```diff
@@ -1379,17 +1381,6 @@ as text, the math role embeds an image. That means:
 
 So, use it only in emergencies ;-)
 
-You can also use an inline substitution of the math directive for things you use often,
-which is the same as using the math role::
-
-    This is the square of x: |xsq|
-
-    .. |xsq| math:: x^2
-
-This is the square of x: |xsq|
-
-.. |xsq| math:: x^2
-
```

Comment:
Why was this removed? 

If we're removing it from the manual, should we remove support from the code too?

Reply:
The replacement syntax for rst2html (that builds the HTML manual with a few additions) differs from the syntax for rst2pdf and I couldn't get a version working that rendered in both cases. I have left the source and removed the example. I'm reasonably convinced that the math support is sufficiently seldom used that a working example of a specific replacement within this area was not worth blocking the HTML manual for!  The feature should be supported and the source showing the example is still in the manual - we just can't render the example in this location in every format.

Reply:
Works for me.